### ISSUE TITLE
pip install fails by not finding psycopg2-binary==2.8.4

### ISSUE THREAD

**Issue by MalteEbner** [heartexlabs/label-studio#1285]
**Describe the bug**
Running `pip install label-studio==1.1.0` in a venv it fails with

```
WARNING: Discarding https://files.pythonhosted.org/packages/00/7b/a623f49b3248957e7eaaac52eba1117209775d54e7a8501c460473a7ba4f/psycopg2-binary-2.8.4.tar.gz#sha256=3a2522b1d9178575acee4adf8fd9f979f9c0449b00b4164bb63c3475ea6528ed (from https://pypi.org/simple/psycopg2-binary/) (requires-python:>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement psycopg2-binary==2.8.4 (from label-studio) (from versions: 2.7.4, 2.7.5, 2.7.6, 2.7.6.1, 2.7.7, 2.8, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.8.5, 2.8.6, 2.9, 2.9.1)
ERROR: No matching distribution found for psycopg2-binary==2.8.4
```

**To Reproduce**
`pip install label-studio==1.1.0`

**Environment (please complete the following information):**
 - OS: macOS 11.4 (20F71)
 - Label Studio Version 1.1.0
 - Python version: 3.9
 - pip version: 21.2.3

**Comment by maxmahe1** [comment]
Hey! Got the same issue but found the solution by creating a new conda environment and installing it in it. Worked fine afterwards. Hope it helps

**Comment by MalteEbner** [comment]
I solved it by using python 3.8 instead.
